ipn/ipnlocal/local.go

```diff
@@ -432,6 +432,7 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 			ss.HostName = b.netMap.Hostinfo.Hostname
 			ss.DNSName = b.netMap.Name
 			ss.UserID = b.netMap.User
+			ss.ExitNodeIP = b.exitNodeIPLocked()
```

Comment:
I don't think this should be on Self (which is of type PeerStatus) at all, because then we have an unused field for all the Peers (which are also of type PeerStatus).